pages/products/join-colleagues.html

```diff
@@ -1,7 +1,9 @@
 ---
 title: "Welcome to the Esteemed Colleagues Community"
 description: "We're here to help you succeed."
-permalink: "/join-talent-network/"
+permalink: "/join-colleagues/"
+redirect_from:
```

Comment:
@alex-novak please note that we can use `redirect_from` instead of `redirect_to` when doing something like this.